handler/core/src/main/java/io/knotx/fragments/handler/FragmentsHandler.java

```diff
@@ -50,15 +51,15 @@
 
   private final FragmentsEngine engine;
   private final TaskProvider taskProvider;
+  private final FragmentsHandlerOptions handlerOptions;
   private final FragmentExecutionLogConsumersNotifier consumerNotifier;
 
   FragmentsHandler(Vertx vertx, JsonObject options) {
-    FragmentsHandlerOptions handlerOptions = new FragmentsHandlerOptions(options);
+    handlerOptions = new FragmentsHandlerOptions(options);
     taskProvider = new TaskProvider(handlerOptions.getTaskFactories(), vertx);
     engine = new FragmentsEngine(vertx);
     requestContextEngine = new DefaultRequestContextEngine(getClass().getSimpleName());
-    consumerNotifier = new FragmentExecutionLogConsumersNotifier(
-        handlerOptions.getConsumerFactories());
+    consumerNotifier = new FragmentExecutionLogConsumersNotifier(handlerOptions.getConsumerFactories());
```

Comment:
Autoformatter wraps this line - please revert this

Reply:
Sure

Btw, what autoformatter? If you mean IntelliJ then it doesn't for me